src/plugins/swagger.ts

```diff
@@ -14,6 +14,11 @@ export default async function (instance: FastifyInstance): Promise<void> {
         description: 'Graasp Backend API used to serve data to frontend applications',
       },
     },
+    refResolver: {
+      buildLocalReference(json, baseUri, fragment, i) {
+        return json.$id?.toString() || `my-fragment-${i}`;
+      },
+    },
```

Comment:
I'm not sure to understand this part, maybe it could be good to add a comment for clarity?